setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -8,6 +8,14 @@ vs.localizedResources
                        title="Microsoft VisualStudio Managed Project System Common Files"
                        description="Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects"
 
+vs.dependencies
+  vs.dependency id=Microsoft.NetCore.Component.DevelopmentTools
+                version=[$(Version),)
```

Comment:
What is $(Version) here referring to?

Reply:
Version is the [VsixVersion](https://github.com/dotnet/project-system/blob/master/setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.csproj#L14)

Reply:
Why does our version determine what versions we have a dependency on?

Reply:
Don't ascribe any meaning here, this is just convenience because we follow the version of the product, so we don't need to keep updating this file.

Reply:
Walk though this:

- We insert a new package from our 16.6 before Roslyn has inserted from 16.6, now we fail to install because our dependencies say we depend on "16.6" of Roslyn which doesn't yet exist.